### DISCUSSION TITLE
version 5.0.0での投稿添付追加時に文字入力できなくなる

### DISCUSSION THREAD

**Discussion by kaias1jp** [tateisu/SubwayTooter#200] [Q&A]
https://user-images.githubusercontent.com/24523508/196298128-22f1a2cf-9c60-46f2-b97d-4821e5c393fc.mp4

投稿画面で画像を添付すると、サーバーへの画像アップロード完了時のメッセージ表示により文字入力が一時的にできなくなります。
メッセージを表示しても文字入力に影響しないように出来ないでしょうか？

**Comment by tateisu** [comment]
https://github.com/tateisu/SubwayTooter/commit/387b532a881d335c0aeb3c03d173f6a0f7792909 で対応済み。まだプレリリースでしたね

**Comment by tateisu** [comment]
- GitHubのリリースページの5.0.3からプレリリースフラグを外した。
- Playストアに5.0.3を製品版チャネルで投げた。
